ios/RCTWebRTC/WebRTCModule+RTCPeerConnection.m

```diff
@@ -377,22 +396,18 @@ - (void)peerConnection:(RTCPeerConnection *)peerConnection didChangeSignalingSta
 }
 
 - (void)peerConnection:(RTCPeerConnection *)peerConnection didAddStream:(RTCMediaStream *)stream {
+  NSString *streamReactTag = [[NSUUID UUID] UUIDString];
   NSMutableArray *tracks = [NSMutableArray array];
   for (RTCVideoTrack *track in stream.videoTracks) {
     peerConnection.remoteTracks[track.trackId] = track;
+    [peerConnection addVideoTrackAdapter:streamReactTag track:track];
     [tracks addObject:@{@"id": track.trackId, @"kind": track.kind, @"label": track.trackId, @"enabled": @(track.isEnabled), @"remote": @(YES), @"readyState": @"live"}];
   }
   for (RTCAudioTrack *track in stream.audioTracks) {
     peerConnection.remoteTracks[track.trackId] = track;
     [tracks addObject:@{@"id": track.trackId, @"kind": track.kind, @"label": track.trackId, @"enabled": @(track.isEnabled), @"remote": @(YES), @"readyState": @"live"}];
   }
 
-  NSString *streamReactTag;
-  // Make sure ID does not exist across local and remote streams (for any peerConnection)
```

Comment:
this goes away, because it's unlikely that the conflict will happen ?

Reply:
More impossible than unlikely, it's a UUIDv4.

Reply:
even if those are generated on different machines (remote streams) ?

Reply:
Yes. https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random) see the last sentence.